### PR TITLE
fix: single add select css updates

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #2014 

fixes some css issues found during the single add select design review

### CODE REVIEW

File: packages/cloud-cognitive/src/components/AddSelect/_add-select.scss [hunk 1]
```diff
@@ -100,6 +100,17 @@
...
   }
 
+  .#{$block-class}__selections-row:hover {
+    .#{$block-class}__selections-radio {
+      color: $interactive-04;
+    }
+
+    .#{$block-class}__selections-view-children svg path {
+      // stylelint-disable-next-line
+      fill: $interactive-04 !important;
```
Comment (matthewgallo) [inline packages/cloud-cognitive/src/components/AddSelect/_add-select.scss:110]: Any way to possibly work around the `!important`?

Comment (davidmenendez) [reply packages/cloud-cognitive/src/components/AddSelect/_add-select.scss:110]: yeah but it would just be a very long selector. but i can switch it.

**Comment by davidmenendez** [comment]
@matthewgallo fixt

**Review by matthewgallo — APPROVED** [review]
🎉

**Pull request merged**